feat: Keep manual changes to the PR body upon update (#19596)
renovatebot/renovate: docs/usage/configuration-templates.md

- Add a footer by using `prFooter`
- Add a note by using `prBodyNotes`
- Edit the embedded table by using `prBodyDefinitions` and `prBodyColumns`

### Renovate tags

> This tags are only available by a subset of the platforms where html tags are supported, e.g., GitHub, GitLab, ...

Reviewer: ```suggestion
Renovate tags are only available on platforms that support HTML tags:

- GitHub
- GitLab
- ...
```

#### renovate:start and renovate:end

> The Renovate specific description in a new pull requests will be wrapped by the tags e.g.,

Reviewer: ```suggestion
The Renovate specific description is wrapped between the `<!!-renovate:start-->` and `<!--renovate:end` tags in the body text of the PR, like this:
```

```md
<!--renovate:start-->

Renovate specific body

<!--renovate:end-->
```

> This allows users to extend the description outside of this tags with custom information without losing the changes in the pull request description upon an automated pull request update by the Renovate Bot. Only the part between the two tags will be replaced with an update Renovate body.

Reviewer: ```suggestion
This allows users to extend the description outside of this tags with custom information without losing the changes in the pull request description upon an automated pull request update by Renovate bot.
Only the part between the two tags will be replaced with an updated Renovate body.
```

> In case only one of the two tags is available, either all text after `renovatge:start` or before `renovate:end` is replaced with the Renovate specific body.

Reviewer: ```suggestion
If only one of the two tags is available then either all text after `renovate:start` or before `renovate:end` is replaced with the Renovate specific body.
```

Author: Thank you for the suggestions. I have now simplified the wording to the bare minimum.
54ca21b3f314c74270ccd28424400b8d2829c819

- Add a footer by using `prFooter`
- Add a note by using `prBodyNotes`
- Edit the embedded table by using `prBodyDefinitions` and `prBodyColumns`

### Renovate tags

Renovate tags are only available on platforms that support HTML tags:

- GitHub
- GitLab

> - ...

Reviewer 2: ```suggestion
- Gitea
```

> - Gitea

Reviewer 2: sort alphanumeric

Author: c3b006a10c3de734be4435cd91a76ad94a02c4cd
